cohortextractor/query_language.py

```diff
@@ -44,6 +56,22 @@ def __or__(self, other):
     def __len__(self):
         return len(self.children)
 
+    def __invert__(self):
+        obj = copy(self)
```

Comment:
Is this `copy()` call doing anything more than calling the `__copy__()` method below? If not, could we just get rid of the `import copy`?

Reply:
It's not, no.  It just means that `copy.copy` on one of these used elsewhere would use that method.  But we don't actually use it elsewhere, so  it's probably better to be more explicit about it